yaeger.spacesimulator/src/main/java/com/yaeger/spacesimulator/ui/entities/controls/VolumeValueControl.java

```diff
@@ -6,8 +6,8 @@
 
 public class VolumeValueControl extends ValueControl {
 
-	public VolumeValueControl(Coordinate2D initialLocation, double width) throws Exception {
-		super(initialLocation, width, "Volume", 1, 1, 1500, new DecimalFormat("###,###,###.###"));
+	public VolumeValueControl(Coordinate2D initialLocation, double width) throws Exception {
+		super(initialLocation, width, "Volume", 20, 20, 200, new DecimalFormat("###,###,###.###"));

```

Comment:
Is het niet een idee, om dit voor 't grote tweaken, om te zetten naar het configbestand? Dan kunnen we alles tweaken vanuit 1 punt. En niet vanuit overal en nergens in de code